source/jormungandr/jormungandr/interfaces/v1/serializer/status.py

```diff
@@ -80,6 +80,32 @@ class TravelerProfilesSerializer(serpy.Serializer):
     walking_speed = Field(schema_type=float)
     wheelchair = Field(schema_type=bool)
 
+class AutocompleteSerializer(serpy.DictSerializer):
+    class_ = Field(schema_type=str, label='class', attr='class')
+    timeout = MethodField(schema_type=float, display_none=False)
+
+    def get_timeout(self, obj):
+        return obj.get('timeout', None)
+
+class CircuitBreakerSerializer(serpy.DictSerializer):
+    current_state = Field(schema_type=str, display_none=True)
+    fail_counter = Field(schema_type=int, display_none=True)
+    reset_timeout = Field(schema_type=int, display_none=True)
+
+class StreetNetworkSerializer(serpy.DictSerializer):
+    class_ = Field(schema_type=str, label='class', attr='class')
+    id = Field(schema_type=str)
+    modes = StringListField(display_none=True)
+    timeout = MethodField(schema_type=float, display_none=False)
+    circuit_breaker = MethodField(schema_type=float, display_none=False)
```

Comment:
schema_type is wrong. And why do you use MethodField everywhere? That's the last ugly fallback.

Reply:
hum, forgot to update it :(

I use method field because serpy cannot serialize an optional attr with a DictSerializer